app/main.py

```diff
@@ -1,13 +1,17 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {"line": error["line_number"],
+            "column": error["column_number"],
+            "message": error["text"],
+            "name": error["code"],
+            "source": "flake8"}
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {"errors": [format_linter_error(error) for error in errors],
```

Comment:
You could use an alternative syntaxis here as well, just as an option:
```suggestion
    return list(map(format_linter_error, errors)),
```

Reply:
done